scripts/env_matrix.yml

```diff
@@ -4,6 +4,7 @@ CONDA_PY:
   - "3.5"
 CONDA_BOOST: "1.60"
 CONDA_PERL: "5.22.0"
+iCONDA_R: "3.3.0"
```

Comment:
`i`?
I think the next version that will hit the `r` channel will be at least 3.3.1